concourse/scripts/deployer-environment.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -euo pipefail
+
+SCRIPT_DIR=$(cd "$(dirname "$0")" && pwd)
+PROJECT_DIR=$(cd "${SCRIPT_DIR}/../.." && pwd)
+
+DEPLOY_ENV=${1:-${DEPLOY_ENV:-}}
+if [ -z "${DEPLOY_ENV}" ]; then
+  echo "Must specify DEPLOY_ENV as \$1 or environment variable" 1>&2
+  exit 1
+fi
+
+AWS_ACCOUNT=${AWS_ACCOUNT:-dev}
+
+CONCOURSE_ATC_USER=${CONCOURSE_ATC_USER:-admin}
+CONCOURSE_URL="${CONCOURSE_URL:-https://concourse.${SYSTEM_DNS_ZONE_NAME}}"
+FLY_TARGET=${FLY_TARGET:-$DEPLOY_ENV}
+FLY_CMD="${PROJECT_DIR}/bin/fly"
+
+CONCOURSE_ATC_PASSWORD=$(aws s3 cp "s3://gds-paas-${DEPLOY_ENV}-state/concourse-secrets.yml" - | grep "concourse_atc_password" | cut -d ':' -f 2 | tr -d ' ')
```

Comment:
Having discussed this I think this is fine to merge as is because this pattern is currently used in several other places. Recent changes to how we generate passwords also mean that they should always be unquoted.